styles.css

```diff
@@ -0,0 +1,1240 @@
+    padding-top: 2mm;
+    background-color: white;
+    display: grid;
+    grid-template: "pic  info     dots"1fr/ 20% 60% 20%;
+}
+
+#three-dots {
+    max-width: 30px;
+    padding-top: 1mm;
+}
+
+#bottom-profile {
+    grid-area: pic;
+
+}
+
+#bottom-profile img {
+    max-width: 100%;
+    max-height: 100%;
+    object-fit: cover;
+    border-radius: 60%;
+}
+
+#bottom-info {
+    grid-area: info;
+    margin-left: 2mm;
+    margin-top: 2mm;
+}
+
+#bottom-info h4 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bolder;
+    font-size: 4mm;
+}
+
+#bottom-info p {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    font-size: 3.5mm;
+    color: #a2a2a2;
+
+}
+
+#three-dots {
+    grid-area: dots;
+    margin-top: 2mm;
+}
+
+#bottom-info:hover {
+    box-shadow: inset 5em 1em rgb(254 252 13);
+}
+
+
+#bottom-profile img:hover {
+    box-shadow: 5px 5px 10px rgb(207 208 209);
+
+}
+
+#three-dots svg:hover {
+    fill: rgb(22 124 192);
+}
+
+.main-header {
+    grid-area: main-header;
+    background-color: white;
+    border-left: 2px solid rgb(204 215 221);
+    border-right: 2px solid rgb(204 215 221);
+    text-align: left;
+    padding-top: 4mm;
+    padding-left: 3mm;
+    display: grid;
+    grid-template-columns: 95% 5%;
+}
+
+.main-header p {
+    font-weight: bolder;
+    font-size: large;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.main-header img {
+    width: 20px;
+    border-radius: 50px;
+}
+
+.main-header img:hover {
+    background-color: rgb(231 231 232);
+}
+
+.middle-main {
+    grid-area: middle-main;
+    background-color: white;
+    border-top: 2px solid rgb(204 215 221);
+    border-left: 2px solid rgb(204 215 221);
+    border-right: 2px solid rgb(204 215 221);
+    padding-bottom: 1mm;
+    min-width: 300px;
+    display: grid;
+    grid-template: "pic  main    main"2fr "pic   icons   tweak"1fr/ 10% 1fr 66px;
+
+}
+
+#middle-profile {
+    grid-area: pic;
+    margin-bottom: 70%;
+    border-radius: 60%;
+    margin-left: 10px;
+    margin-top: 2mm;
+}
+
+
+#middle-profile img {
+    width: 50px;
+    border-radius: 60%;
+}
+
+#middle-happening {
+    grid-area: main;
+    padding-top: 7mm;
+    font-weight: 500;
+    font-size: larger;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+#middle-icons {
+    grid-area: icons;
+}
+
+#media,
+#gif,
+#poll,
+#emoji,
+#schedule {
+    width: 20px;
+    display: inline-block;
+    padding-top: 2mm;
+    margin-right: 4mm;
+}
+
+#media:hover {
+    box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, .2);
+    border-radius: 100px;
+}
+
+#gif:hover {
+    box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, .2);
+    border-radius: 100px;
+}
+
+#poll:hover {
+    box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, .2);
+    border-radius: 100px;
+}
+
+#emoji:hover {
+    box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, .2);
+    border-radius: 100px;
+}
+
+#schedule:hover {
+    box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, .2);
```

Comment:
A lot of these styles look to have the same rules (schedule, emoji, poll, gif, media). It would be better to do this as one style and apply it to each of the elements. That way you don't have to repeat the styles you've already written.